### PR TITLE
fix group by clause to be be compliant with mysql 5.7

### PR DESCRIPTION
**Pull request by FliegenKLATSCH**
In mysql 5.7 only_full_group_by is enabled by default.

### CODE REVIEW

File: install/froxlor.sql [hunk 1]
```diff
@@ -66,7 +66,7 @@ CREATE TABLE `mail_virtual` (
   `id` int(11) NOT NULL auto_increment,
   `email` varchar(255) NOT NULL default '',
   `email_full` varchar(255) NOT NULL default '',
-  `destination` text NOT NULL,
+  `destination` text DEFAULT NULL,
```
Comment (d00p) [inline install/froxlor.sql]: you are absolutely right

**Comment by FliegenKLATSCH** [comment]
Hmm.. I think I got an error, because there was no default value..
So I will change the default from null to empty string? I don't see the point here, but I will change it, probably tomorrow..

**Comment by nachtgeist** [comment]
> Hmm.. I think I got an error, because there was no default value..

Ah I see.

> So I will change the default from null to empty string?

ACK

**Comment by FliegenKLATSCH** [comment]
Done.. Anything else?

**Pull request merged**